quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testImportHelpAll.approved.txt

```diff
@@ -266,18 +266,18 @@ Tracing:
                        the 'tracing-resource-attributes' property. Default: keycloak. Available
                        only when Tracing is enabled.
 
-Events (Preview):
+Events:
 
 --event-metrics-user-enabled <true|false>
-                     Preview: Create metrics based on user events. Default: false. Available only
-                       when metrics are enabled and feature user-event-metrics is enabled.
+                     Create metrics based on user events. Default: false. Available only when
+                       metrics are enabled and feature user-event-metrics is enabled.
 --event-metrics-user-events <events>
-                     Preview: Comma-separated list of events to be collected for user event
-                       metrics. Reduce the number of metrics. If empty or not set, all events
-                       create a metric. Possible values are:<...>
+                     Comma-separated list of events to be collected for user event metrics. Reduce
```

Comment:
The sentence `Reduce the number of metrics.` in the middle of the description does not make much sense to me. Is it correct?

Reply:
When the user metrics are published we publish a metric for each event from the list. This option can be used to reduce the number of metrics by specifying what events are published. Are you suggesting removing the sentence?

Maybe we can rephrase to `Can be used to reduce the number of metrics created as by default all events create a metric.`

Reply:
`This option can be used to reduce the number of metrics by specifying what events are published`
I wouldn't have guessed it :) 
+1 to the rephrase.